ts/doc.go

```diff
@@ -0,0 +1,139 @@
+timestamped data points, and efficiently stores it in the key-value store.
+
+Storing time series data is a unique challenge for databases. Time series data
+is typically generated at an extremely high volume, and is queried by providing
+a range of time of arbitrary size, which can lead to an enormous amount of data
+being scanned for a query. Many specialized time series databases exist to meet
+these challenges.
+
+Cockroach organizes time series data by series name and timestamp; when querying
+a given series over a time range, all data in that range is thus sorted
+contiguously. However, cockroach transforms the data in a few important ways in
+order to effectively query a large amount of data.
+
+###Downsampling
+
+The amount of data produced by time series sampling can be staggering; the naive
+solution, storing every incoming data point with perfect fidelity in a unique
+key, would commmand a tremendous amount of computing resources.
+
+However, in most cases a perfect fidelity is not necessary or desired; the exact
+time a sample was taken is unimportant, with the overall trend of the data over
+time being far more important to analysis than the individual samples.
+
+With this in mind, Cockroach downsamples data before storing it; the original
+timestamp for each data point in a series is not recorded. Cockroach instead
+divides time into contiguous slots of uniform length (e.g. 10 seconds); if
+multiple data points for a series fall in the same slot, they are aggregated
+together.  Aggregated data for a slot records the count, sum, min and max of
+data points that were found in that slot. A single slot is referred to as a
+"sample", and the length of the slot is known as the "sample duration".
+
+Cockroach also uses its own key space efficiently by storing data for
+multiple samples in a single key. This is done by again dividing time into
+contiguous slots, but with a longer duration; this is known as the "key
+duration". For example, cockroach samples much of its internal data at a
+resolution of 10 seconds, but stores it with a "key duration" of 1 hour, meaning
+that all samples that fall in the same hour are stored at the same key. This
+strategy helps reduce the number of keys scanned during a query.
+
+Finally, cockroach can record the same series at multiple sample durations, in a
+process commonly known as a "rollup".  For example, a single series may be
+recorded with a sample size of 10 seconds, but also record the same data with a
+sample size of 1 hour. The 1 hour data will have much less information, but can
+be queried much faster; this is very useful when querying a series over a very
+long period of time (e.g. an entire month or year).
+
+A specific sample duration in cockroach is known as a Resolution. Cockroach
+supports a fixed set of Resolutions; each Resolution has a fixed sample duration
+and a key duration. For example, the resolution "Resolution10s" has a sample
```

Comment:
It might, but Resolution10s1h looked kind of odd to me... I'll make this decision when we introduce another resolution.